.gitignore

```diff
@@ -1 +1,4 @@
-.vscode
\ No newline at end of file
+.vscode
+.idea
+eureka-cli/misc/folio-keycloak
+eureka-cli/misc/folio-kong
```

Comment:
See my comment below about these being submodules.

Reply:
Yes, these were also excluded on the yet unmerged `dev` branch: <https://github.com/folio-org/eureka-setup/blob/dev/eureka-cli/.gitignore>